benchmarks/rectangles-area-difference.bril

```diff
@@ -0,0 +1,19 @@
+# ARGS: 5 10 6 13
+@main (x1 : int, y1: int, x2: int, y2 : int) {
+      a1: int = call @area x1 y1;
+      a2: int = call @area x2 y2;
+      res: int = sub a1 a2;
+      a1_bigger: boolean = gt a1 a2;
```

Comment:
Did you mean: `boolean` -> `bool`?

Reply:
Ahh sorry, it seems like I missed the syntax a bit. But weirdly enough `boolean` also works? (I tested both ways locally) I'll make a PR with the fix, thanks for pointing this out!

Reply:
Yeah it's just #155. It's technically fine for `brili` but I've been trying to keep `brilirs` working on all of the benchmarks and it more strictly follows the specification.